notebooks/community/sdk/SDK_BigQuery_Custom_Container_Training.ipynb

```diff
@@ -58,8 +58,15 @@
       },
```

Comment:
<p>Heading, use sentence case</p><p>Use only 2nd person (we -&gt; you)</p><p>Do not use future tense.</p><p> </p><p>We will make a&nbsp; -&gt; First, you make a</p><p> </p><p>Big Query -&gt; BigQuery</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/849/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>